src/constants/auth.js

```diff
@@ -16,6 +16,11 @@
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+// Length of the random string for login
 export const tokenNameLength = 40;
+
+// By default the scope for user is to write
 export const tokenScope = "write";
-export const tokenExpiryDays = 2;
+
+// Authorization token expiry days
+export const tokenExpiryDays = 1;
```

Comment:
Just a question, would `.env` be a better place for these constants? And have same default value if `.env` does not exists?

Reply:
Yes sure, we can do it

Reply:
So can you please create an issue for that and do it in a new PR?

Reply:
Okay sure.